packages/core/src/utils/theme.ts

```diff
@@ -127,6 +127,7 @@ export const createTheme = (
   }
 
   // Created theme
+  customizedTheme.base = base;
```

Comment:
We already have this (`customizedTheme.base = base`) in the `createTheme` on line 103. We can remove this one.

Reply:
Right, remains from previous work :)